libass/ass_shaper.c

```diff
@@ -860,6 +858,13 @@ void ass_shaper_set_level(ASS_Shaper *shaper, ASS_ShapingLevel level)
     shaper->shaping_level = level;
 }
 
+#ifdef USE_FRIBIDI_EX_API
+void ass_shaper_set_bidi_brackets(ASS_Shaper *shaper, bool match_brackets)
+{
+    shaper->bidi_brackets = !!match_brackets;
```

Comment:
I think that `!!` is redundant here as both `bidi_brackets` and `match_brackets` are bools.

Reply:
Removed the `!!`.